proto/dydxprotocol/clob/tx.proto

```diff
@@ -98,6 +101,33 @@ message MsgCancelOrder {
 // MsgCancelOrderResponse is a response type used for canceling orders.
 message MsgCancelOrderResponse {}
 
+// MsgBatchCancel is a request type used for batch canceling orders.
```

Comment:
actually should we just call this MsgBatchShortTermCancel because later on we'll probably make one for Stateful?

Reply:
I think MsgBatchCancel is fine, since the internal field is already called `short_term_cancels` so `MsgBatchShortTermCancel` feels a little bit redundant.

also how we want to support stateful batch cancel is probably worth another discussion (maybe we can overload this so that users don't need two separate txns).